openpype/settings/entities/schemas/projects_schema/schema_project_fusion.json

```diff
@@ -68,6 +68,50 @@
                     "label": "Resync profile on each launch"
                 }
             ]
+        },
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "create",
+            "label": "Creator plugins",
+            "children": [
+                {
+                    "type": "dict",
+                    "collapsible": true,
+                    "key": "CreateSaver",
+                    "label": "Create Saver",
+                    "is_group": true,
+                    "children": [
+                        {
+                            "type": "text",
+                            "key": "temp_rendering_path_template",
+                            "label": "Temporary rendering path template"
+                        },
```

Comment:
Temporary rendering path feels like an odd wording for this. I can't think of anything else but it wasn't until I actually read what the default value is for this key that I understood what this tried to refer to. Can we think of anything that fits better?

Reply:
I am having no better name for now. This is now aligned with Nuke. But will add TODO note for later rename.